src/Symfony/State/ServiceLocatorProcessor.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Symfony\State;
+
+use ApiPlatform\Metadata\AbstractOperation;
+use ApiPlatform\State\ProcessorInterface;
+
+class ServiceLocatorProcessor implements ProcessorInterface
```

Comment:
```suggestion
final class ServiceLocatorProcessor implements ProcessorInterface
```

And as this class doesn't depend of Symfony, it could be placed in the top-level namespace.